bibliopixel/project/project.py

```diff
@@ -62,7 +64,11 @@ def make_remote(layout, remote):
         run = anim.get('run', {})
         run['threaded'] = True  # Remote anims must be threaded, maybe a little hacky?
         anim_obj = dict(anim)
-        anim_obj['animation'] = make_animation(layout, anim['animation'], run)
+        try:
+            anim_obj['animation'] = make_animation(layout, anim['animation'], run)
+        except:
+            log.error(traceback.format_exc())
+            anim_obj['animation'] = None
```

Comment:
Urg, not so happy about this.  Right now, none of our configuration dictionaries have `None` in them, which means if you `get` a value and it's `None` you know it didn't exist.

What about either assigning the value to be `{}` or deleting the value from the dictionary?